.travis.yml

```diff
@@ -2,12 +2,11 @@ language: python
 python:
 - '2.7'
 install:
-- pip install -r requirements.txt
-- pip install -r requirements_test.txt
+- pip install tox
+- pip install .
```

Comment:
Is this line necessary? Running tox should install the local app as part of its virtualenv setup.